wp-content/themes/csisnuclearnetwork/assets/_scss/pages/home.scss

```diff
@@ -493,4 +493,125 @@
       }
     }
   }
+
+  .newsletter-block-acf {
+    display: grid;
+    grid-template-areas: 
+      'header'
+      'daily'
+      'monthly';
+    padding-top: rem(56);
+    padding-bottom: rem(72);
+    @include full-width-background(linear-gradient(180deg, #153b6a 0%, #215f8c 100%));
+    background: linear-gradient(180deg, #153b6a 0%, #215f8c 100%);
+
+    @include breakpoint(medium) {
+      grid-template-areas: 
+        'header header'
+        'daily monthly';
+      column-gap: rem(40);
+    }
+
+    @include breakpoint(large) {
+      grid-template-areas: 'header daily monthly';
+    }
+
+    .home__subtitle {
+      --home-subtitle-color: #{$color-white-100};
+      grid-area: header;
+    }
+
+    p {
+      flex-basis: 100%;
+      margin-top: rem(8);
+      margin-bottom: rem(24);
+      @extend %text-style-paragraph-large;
+      @extend %text-style-short;
```

Comment:
Instead of using `@extend %text-style-short`, I would add `text--short` to the `<p>` tag as a class instead.

Reply:
got it